activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Bump minimum MySQL version to 5.7.9.
+
+    *Yasuo Honda*
+
+*   Use MySQL utf8mb4 character set by default.
+
+    *Yasuo Honda*
+
```

Comment:
Let's explain it a little so people understand why we've made the change.

Reply:
Sure. Let me have some more time to update the changelog entry.

Reply:
Added some more entries why we need utf8mb4 character set.